module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -412,22 +413,26 @@ public function getAuthMethod()
      * Get the name of the currently selected authentication method (if applicable)
      * or the active authentication method.
      *
-     * @return string
+     * @return ?string
      */
-    public function getSelectedAuthMethod()
+    public function getSelectedAuthMethod(): ?string
     {
         $auth = $this->getAuth();
-        return is_callable([$auth, 'getSelectedAuthOption'])
+        $selectedAuthMethod = is_callable([$auth, 'getSelectedAuthOption'])
             ? $auth->getSelectedAuthOption()
             : $this->getAuthMethod();
+        if ($selectedAuthMethod === false) {
```

Comment:
Would a less strict comparison be better here, like `if (!$selectedAuthMethod) {` ? If authentication handlers aren't strictly typed, I imagine different false-y values might slip through here, and I don't see a disadvantage to being flexible about what gets normalized to null.